adhocteam/Head-Start-TTADP
# improve dashboard button labeling

## Description of change

Per recommendation from Lauren & our accessibility specialist, the language on the "show accessible data" buttons was revised to be "display graph" and "display table." While making these revisions, language was added to distinguish the aria-labels on the "apply filters" and "open menu" buttons.

## How to test

Visit the dashboard. View the revised language on the buttons. Using a screen-reader or element inspector. Note that the aria-labels for CheckboxSelect and ButtonSelect components has changed.

## Issue(s)

## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
